Include/cpython/context.h

```diff
@@ -27,6 +27,43 @@ PyAPI_FUNC(PyObject *) PyContext_CopyCurrent(void);
 PyAPI_FUNC(int) PyContext_Enter(PyObject *);
 PyAPI_FUNC(int) PyContext_Exit(PyObject *);
 
+#define Py_FOREACH_CONTEXT_EVENT(V) \
+   V(ENTER)                      \
+   V(EXIT)
```

Comment:
Should these be exposed in Python.h, or are they only for internal use?

Reply:
this is the same as what is done with PY_FOREACH_CODE_EVENT its used interanal.

Reply:
If it is internal we should not expose it. IMO, `PY_FOREACH_CODE_EVENT` should not have been exposed in Python.h.

Reply:
@erlend-aasland is putting a _ infront of it enough or do I need to #undef it somewhere

Reply:
You can put the internal APIs in `Include/internal/pycore_context.h`; everything in `Include/internal/*` is excluded from Python.h.